phpmyadmin/content.md

```diff
@@ -6,74 +6,113 @@ Run phpMyAdmin with Alpine, Apache and PHP FPM.
 
 The following tags are available:
 
 -	`latest`, `fpm`, and `fpm-alpine` are always the most recent released version
 -	Major versions, such as `5`, `5-fpm`, and `5-fpm-alpine`
 -	Specific minor versions, such as `5.0`, `5.0-fpm`, and `5-fpm-alpine`
--	Specific patch versions, such as `5.0.0`, `5.0.0-fpm`, and `5.0.0-alpine`. Note that, on rare occasion, here may be an intermediary "docker-only" release, such as 4.9.2-1
+-	Specific patch versions, such as `5.0.0`, `5.0.0-fpm`, and `5.0.0-fpm-alpine`. Note that, on rare occasion, there may be an intermediary "docker-only" release, such as 4.9.2-1
+
+A complete list of tags is [available at Docker Hub](https://hub.docker.com/_/phpmyadmin?tab=tags)
+
+## Image variants
+
+We provide three variations:
+
+-	"apache" includes a full Apache webserver with PHP and includes everything needed to work out of the box. This is the default when only a version number is requested.
+-	"fpm" only starts a PHP FPM container. Use this variant if you already have a separate webserver. This includes more tools and is therefore a larger image than the "fpm-alpine" variation.
+-	"fpm-alpine" has a very small footprint. It is based on Alpine Linux and only starts a PHP FPM process. Use this variant if you already have a separate webserver. If you need more tools that are not available on Alpine Linux, use the fpm image instead.
 
 ## Usage with linked server
 
-First you need to run MySQL or MariaDB server in Docker, and this image need link a running mysql instance container:
+First you need to run a MySQL or MariaDB server in Docker, and the phpMyAdmin image needs to be linked to the running database container:
 
-```console
-$ docker run --name myadmin -d --link mysql_db_server:db -p 8080:80 %%IMAGE%%
+```sh
+docker run --name phpmyadmin -d --link mysql_db_server:db -p 8080:80 %%IMAGE%%
 ```
 
 ## Usage with external server
 
-You can specify MySQL host in the `PMA_HOST` environment variable. You can also use `PMA_PORT` to specify port of the server in case it's not the default one:
+You can specify a MySQL host in the `PMA_HOST` environment variable. You can also use `PMA_PORT` to specify the port of the server in case it's not the default one:
 
-```console
-$ docker run --name myadmin -d -e PMA_HOST=dbhost -p 8080:80 %%IMAGE%%
+```sh
+docker run --name phpmyadmin -d -e PMA_HOST=dbhost -p 8080:80 %%IMAGE%%
 ```
 
 ## Usage with arbitrary server
 
-You can use arbitrary servers by adding ENV variable `PMA_ARBITRARY=1` to the startup command:
+You can use arbitrary servers by adding the environment variable `PMA_ARBITRARY=1` to the startup command:
 
-```console
-$ docker run --name myadmin -d -e PMA_ARBITRARY=1 -p 8080:80 %%IMAGE%%
+```sh
+docker run --name phpmyadmin -d -e PMA_ARBITRARY=1 -p 8080:80 %%IMAGE%%
 ```
 
 ## Usage with docker-compose and arbitrary server
 
-This will run phpMyAdmin with arbitrary server - allowing you to specify MySQL/MariaDB server on login page.
-
-%%STACK%%
```

Comment:
If you replace the `%%STACK%%`, with an inline yaml then the cute "Play-with-Docker" button will be gone too. Using the `%%STACK%%` keyword will allow the "templating" to drop in the [`stack.yml`](https://github.com/docker-library/docs/blob/5e022ff391ac1b4da2cea04a235f35230940f671/phpmyadmin/stack.yml) along with the play-with-docker button. Unfortunately the `stack.yml` can't have any of our templating since it needs to be passed as a url to play-with-docker.

Reply:
It should be okay, thank you for explaining this so nicely